src/cls/otp/cls_otp_client.cc

```diff
@@ -60,10 +60,9 @@ namespace rados {
         cls_otp_check_otp_op op;
         op.id = id;
         op.val = val;
-#define TOKEN_LEN 16
-        char buf[TOKEN_LEN + 1];
```

Comment:
granted, I may not fully understand the problematic, but does this count as a VLA? I'd assume `buf`'s size would be known at compile time in this case, and most definitely not of variable length.

Reply:
@jecluis Good eyes! This is (I believe the only) a case that's not a VLA. In this case, I saw that an extra copy of "buf" could be avoided in favor of a string construction that can be elided. That said, it might have been overzealous of me. I can go either way on this one-- 16 bytes isn't a large copy and there would obviously be no free store allocation for the array. Line 66, "op.token = buf" is the culprit: this assigns a new value to op.token by copying (C style) from buf. However, it may be that the compiler's smart enough that there's no difference in this case; I'd have to see the actual assembly.